spec/helpers/shell_spec.rb

```diff
@@ -21,7 +21,7 @@ class FakeShell
         Tempfile.open("logfile.log") do |f|
           ENV["BUILDPACK_LOG_FILE"] = f.path
           FakeShell.new.mcount "foo"
-          expect(File.read(f.path)).to match("count#foo=1")
+          expect(File.read(f.path)).to match("count#buildpack.ruby.foo=1")
```

Comment:
Oh it's used for testing... Hmmm 🤔

Reply:
This is the log file, not the prefix, if I hardcode the prefix this test would still pass.

Reply:
🙈 That seems obvious now